util/libcrypto.num

```diff
@@ -4741,7 +4741,7 @@ OSSL_CRMF_CERTTEMPLATE_get0_serialNumber 4688	3_0_0	EXIST::FUNCTION:CMP
 OSSL_CRMF_CERTTEMPLATE_get0_issuer      4689	3_0_0	EXIST::FUNCTION:CMP
 OSSL_CRMF_CERTTEMPLATE_fill             4690	3_0_0	EXIST::FUNCTION:CMP
 OSSL_CRMF_ENCRYPTEDVALUE_get1_encCert   4691	3_0_0	EXIST::FUNCTION:CMP
-ERR_load_CRMF_strings                   4692	3_0_0	EXIST::FUNCTION:CMP
+ERR_load_CRMF_strings                   4692	3_0_0	EXIST::FUNCTION:
```

Comment:
Again, why does this happen?

Reply:
There is no configuration option to disable CMP.  There is one to disable CRMF.  I think mkerr.pl is sensible to this sort of thing.  That should be investigated.

Reply:
This issue has been addressed in #8897.
Therefore we have rebased this PR to the one by @levitte.